cpp/lazperf/vlr.hpp

```diff
@@ -155,6 +155,66 @@ struct LAZPERF_EXPORT wkt_vlr : public vlr
     virtual vlr_header header() const;
 };
 
+
+struct LAZPERF_EXPORT copc_extents_vlr : public lazperf::vlr
+{
+public:
+
+    struct LAZPERF_EXPORT CopcExtent
+    {
+        double minimum;
+        double maximum;
```

Comment:
Might be better to initialize the values?
```suggestion
        double minimum{};
        double maximum{};
```

Reply:
I will add this. There are many structs in that file that aren't initialized. Maybe we should do so @abellgithub?

Reply:
@abellgithub It's what most people would expect as it's the default initializer. Also easier to work with for tests etc. Also it doesn't cost anything.

Reply:
I think the default constructor should be changed to 0 initialization personally. It makes it pretty clear that the value either hasn't been initialized or has no range. The downstream clients can do their necessary logic to compute the min/max. If a value of +- double_max gets stored in the VLR and consumed by a client it can cause unnecessary headache.

As a side note:

```
    minimum((std::numeric_limits<double>::max)()),
    maximum((std::numeric_limits<double>::min)())
```

I believe this should be:

```
    minimum((std::numeric_limits<double>::max)()),
    maximum(-(std::numeric_limits<double>::max)())
```

We ran into this issue, double::min is actually the smallest positive double value.

Reply:
So, I would agree the default constructor should be removed, and we should keep the min/max initialization as 0,0.

Reply:
my latest push does this.